tools/customization_dataset_preparation/NeMo_LLM_Dataset_Preparation_Tutorial.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
This should be just a README.rst file. Why is this a notebook? Also there is no ``dataset_validation.py`` file

Reply:
Currently set to the run instructions in the header of customization_dataset_preparation.py --> this would be much easier for people to see and read when they use the py file separately (w/o needing to download nemo or going to nemo docs).